src/game/game_init.c

```diff
@@ -662,7 +663,9 @@ void init_controllers(void) {
 #ifdef EEP
     // strangely enough, the EEPROM probe for save data is done in this function.
     // save pak detection?
-    gEepromProbe = osEepromProbe(&gSIEventMesgQueue);
+    gEepromProbe = IS_VC()
```

Comment:
Same deal as my comment in save_file.c

Reply:
gIsVC is moved to very early in boot as it is not dependent on N64 graphics stuff. IS_VC() usage is removed.